ocaml/xapi/storage_migrate.ml

```diff
@@ -300,7 +300,9 @@ let copy' ~task ~dbg ~sr ~vdi ~url ~dest ~dest_vdi =
 				);
 			)
 			(fun () -> 
-				Remote.DP.destroy ~dbg ~dp:remote_dp ~allow_leak:false);
+				Remote.DP.destroy ~dbg ~dp:remote_dp ~allow_leak:false;
+				State.remove id State.active_copy
```

Comment:
Where is this state set? It would have been nice if this were symmetrical: set it up above, remove it here. Since it is not, maybe a comment would be helpful where this is coming from. In terms of change, this looks fine.